processor/src/it/java/com/linecorp/decaton/processor/TestUtils.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.decaton.processor;
+
+import java.util.Properties;
+import java.util.concurrent.CountDownLatch;
+
+import org.apache.kafka.clients.producer.ProducerConfig;
+
+import com.google.protobuf.MessageLite;
+
+import com.linecorp.decaton.client.DecatonClient;
+import com.linecorp.decaton.processor.SubscriptionStateListener.State;
+import com.linecorp.decaton.processor.runtime.ProcessorSubscription;
+import com.linecorp.decaton.processor.runtime.RetryConfig;
+import com.linecorp.decaton.processor.runtime.SubscriptionBuilder;
+import com.linecorp.decaton.protobuf.ProtocolBuffersSerializer;
+
+public class TestUtils {
+
+    public static <T extends MessageLite> DecatonClient<T> client(String topic,
```

Comment:
There should be a mechanism to ensure you don't reuse the same topic in different tests, since the `KafkaCluster` is not reset between tests

Reply:
Is random topic name (https://github.com/line/decaton/pull/18/files#diff-2f99db7e997743429863e8a0538f07c2R44) not enough?